examples/basic/main.tf

```diff
@@ -1,7 +1,11 @@
 provider "azurerm" {
   storage_use_azuread = true
 
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false # The feature flag can be removed after clean-up.
+    }
+  }
```

Comment:
These example configurations don't create resource groups, so I don't think this is necessary.

Reply:
I think this is something left behind after a previous test run with the old tests, so it's not supposed to be there 😁 Good catch 👍🏼